src/to_value.cpp

```diff
@@ -9,8 +9,6 @@ namespace Sass {
     // throw a runtime error if this happens
     // we want a well defined set of possible nodes
     throw std::runtime_error("invalid node for to_value");
-    // mute warning
```

Comment:
Interestingly there seems to have been a reason for these superflous returns ...